src/lib/pages/frontsite/Documents/DocumentDetails/DocumentPanel/DocumentTitle.js

```diff
@@ -19,12 +21,38 @@ export class DocumentTitle extends Component {
     }
   }
 
+  getLinkTo(relation) {
+    return relation.pid_type === 'docid'
+      ? FrontSiteRoutes.documentDetailsFor(relation.pid_value)
+      : FrontSiteRoutes.seriesDetailsFor(relation.pid_value);
+  }
+
+  relationsTitle(relationsData) {
+    return !_isEmpty(relationsData) ? (
+      <Header.Subheader>
+        {relationsData.map((rel, index) => (
+          <>
+            <Link key={rel.pid_value} to={this.getLinkTo(rel)}>
+              {rel.record_metadata.title}
+            </Link>
+            {index === relationsData.length - 1 ? ' featuring:' : ' and '}
+          </>
+        ))}
+      </Header.Subheader>
+    ) : null;
+  }
+
   render() {
     const { metadata } = this.props;
     const volume = _get(metadata, 'relations.multipart_monograph[0].volume');
 
     return (
       <>
+        <Header>
```

Comment:
if you don't check if the volume number and relation metadata exists before rendering the Header component, you will most likely end up with an empty space on top of the column (for regular books without multipart and serials)